pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -14,34 +13,68 @@ This section describes how to upgrade your Kommander Management cluster and all
 
 ## Prerequisites
 
-- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- [Download][download_binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.
+- For air-gapped environments only: Download the [images bundle][images_bundle], [charts bundle][charts_bundle] and [Catalog Application charts bundle][cat_apps_bundle].  
+
+## For air-gapped environments only: Load the Docker images into your Docker registry
+
+1.  Download the image bundle file:
```

Comment:
I think this is already documented elsewhere, so you should take this download out and just point to the existing topic.